src/InstagramScraper/Instagram.php

```diff
@@ -63,9 +64,11 @@ class Instagram
      *
      * @return Instagram
      */
-    public static function withCredentials($username, $password, $cache)
+    public static function withCredentials($username, $password)
     {
-        static::$instanceCache = $cache;
+        if(static::$instanceCache == null){
+            static::$instanceCache = new Psr16Adapter('Files');
+        }
```

Comment:
@amirsarhang So how to add custom path for the cache files?

Reply:
@congkv Because we don't use custom path in our project I didn't think about that, thanks for inform me.
but actually I don't know how can we set custom path, I appreciate if you tell me should we do something like this? :

```php
static::$instanceCache = new Psr16Adapter('/set/path');
```
OR
```php
static::$instanceCache = '/set/path';
```
Which one is correct way?

Reply:
@amirsarhang
```php
static::$instanceCache = new Psr16Adapter('Files', new \Phpfastcache\Config\ConfigurationOption([
    'path' => '/set/path'
]));
```